sdk/notif.go

```diff
@@ -69,5 +79,53 @@ Branch : {{.git.branch | default "n/a"}}`,
 	UserNotificationTemplateMap = map[string]UserNotificationTemplate{
 		EmailUserNotification:  UserNotificationTemplateEmail,
 		JabberUserNotification: UserNotificationTemplateJabber,
+		VCSUserNotification: UserNotificationTemplate{
+			Body: DefaultWorkflowNodeRunReport,
+		},
 	}
 )
+
+const DefaultWorkflowNodeRunReport = `[[- if .Stages ]]
+CDS Report [[.WorkflowNodeName]]#[[.Number]].[[.SubNumber]] [[ if eq .Status "Success" -]] ✔ [[ else ]][[ if eq .Status "Fail" -]] ✘ [[ else ]][[ if eq .Status "Stopped" -]] ■ [[ else ]]- [[ end ]] [[ end ]] [[ end ]]
+[[- range $s := .Stages]]
+[[- if $s.RunJobs ]]
+* [[$s.Name]]
+[[- range $j := $s.RunJobs]]
+  * [[$j.Job.Action.Name]] [[ if eq $j.Status "Success" -]] ✔ [[ else ]][[ if eq $j.Status "Fail" -]] ✘ [[ else ]][[ if eq $j.Status "Stopped" -]] ■ [[ else ]]- [[ end ]] [[ end ]] [[ end ]]
+[[- end]]
+[[- end]]
+[[- end]]
+[[- end]]
+
+[[- if .Tests ]]
+[[- if gt .Tests.TotalKO 0]]
+Unit Tests Report
+
+[[- range $ts := .Tests.TestSuites]]
+* [[ $ts.Name ]]
+[[range $tc := $ts.TestCases]]
+  [[- if or ($tc.Errors) ($tc.Failures) ]]  * [[ $tc.Name ]] ✘ [[- end]]
+[[end]]
+[[- end]]
+[[- end]]
+[[- end]]
+`
+
+func (nr WorkflowNodeRun) Report() (string, error) {
```

Comment:
We let the user access all the data for the node run, is there private data inside ?

Reply:
already done I made a custom struct with specifics data